benchmarks/bench_ml.py

```diff
@@ -301,14 +313,14 @@ def evaluate_optimizer(surrogate, model, dataset, n_calls, random_state):
     # below seed is necessary for processes which fork at the same time
     # so that random numbers generated in processes are different
     np.random.seed(random_state)
-    problem = MLBench(model, dataset)
+    problem = MLBench(model, dataset, random_state)
     space = problem.space
 
     # initialization
     estimator = surrogate(random_state=random_state)
     dimensions_names = sorted(space)
     dimensions = [space[d][0] for d in dimensions_names]
-    solver = Optimizer(dimensions, estimator, random_state=random_state)
+    solver = Optimizer(dimensions, estimator, random_state=random_state, acq_optimizer='sampling', acq_func='EI')
```

Comment:
Yup, as @glouppe suggested, I used *_minimize methods, which make it quite simple with setting up the right parameters.